resolvers/deno/npm/managed/common.rs

```diff
@@ -20,7 +23,7 @@ pub struct NpmPackageFsResolverPackageFolderError(deno_semver::StackString);
 
 /// Part of the resolution that interacts with the file system.
 #[async_trait(?Send)]
-pub trait NpmPackageFsResolver: Send + Sync {
+pub trait NpmPackageFsResolver: MaybeSend + MaybeSync {
```

Comment:
Also included some fixes to get this crate compiling when not using the `sync` feature.

Reply:
So supersedes https://github.com/denoland/deno/pull/27409?

Reply:
Oh. Yes.